src/components/Body.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Main from './Main';
+import Side from './Side';
+
+function Body() {
```

Comment:
Lets rename this  component to something more descriptive like "Feed".

Reply:
Done!